docs/get_started.ipynb

```diff
@@ -80,11 +80,41 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 3,
       "metadata": {
         "id": "_wqSAZExy6xV"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "debug": {
+            "argv": [
+              "/export/hda3/borglet/remote_hdd_fs_dirs/0.colab_kernel_py3_nkovela.kernel.nkovela.2707279999023.14b334fb3717c109/mount/server/notebook.par",
+              "kernel",
+              "-f",
+              "/tmp/ipy-be-hphvjaxl/profile_colab/security/kernel-4b9d0a95-ce46-4d79-9bcd-7217e06076ad.json",
+              "--profile-dir",
+              "/tmp/ipy-be-hphvjaxl/profile_colab",
+              "--profile=colab",
+              "--ipython-dir=/tmp/ipy-be-vrvxubir",
+              "--no-secure"
+            ],
+            "build": "Built on Tue Sep 13 06:02:56 2022 (1663074176)\nBuilt by colaboratory-release-builder@lflt2.prod.google.com:/google/src/cloud/buildrabbit-username/buildrabbit-client/google3\nBuilt as //research/colab/notebook:notebook_backend_py3\nBuild ID: dc272e53-f9ff-4991-9d28-698b0397c5c7\nBuilt from changelist 473976297 in a mint client based on //depot/google3\nBuild label: colab-20220913-060100-RC00\nBuild platform: gcc-4.X.Y-crosstool-v18-llvm-grtev4-k8\nBuild tool: Blaze, release blaze-2022.08.25-4 (mainline @469755128)\nBuilt with par options [--compress, --strip]\nCurrently running under Python 3.9.14: embedded.\n",
+            "user": "nkovela"
+          },
+          "ename": "ModuleNotFoundError",
+          "evalue": "ignored",
+          "output_type": "error",
+          "traceback": [
+            "\u001b[0;31mModuleNotFoundError:\u001b[0m No module named 'tensorflow'",
+            "",
+            "\nConsider using a custom runtime: go/colab_binary",
```

Comment:
Non Googlers won't be able to view this link, is there an OSS alternative you could point them towards?

Reply:
This was simply part of an error message when running locally because I didn't have tensorflow installed (forgot to clear my output). Sorry, fixed.

Reply:
I thought that might be the case 👍